src/mutations/auth.mutation.ts

```diff
@@ -34,3 +34,15 @@ export const RESET_PASSWORD_MUTATION = gql`
     resetPassword(data: $data)
   }
 `;
+
+export const CHANGE_PASSWORD_MUTATION = gql`
+  mutation changePassword(
+    $new_password: String!
+    $new_password_confirmation: String!
+  ) {
+    changePassword(
+      new_password: $new_password
+      new_password_confirmation: $new_password_confirmation
+    )
+  }
+`;
```

Comment:
**Question**: you are not passing the old password here that you are using on the method, it works without it?

Reply:
if you see the kanvas-ecosystem code it doesn't grab the old_password anywhere 
![image](https://github.com/bakaphp/kanvas-core-js/assets/40866642/70298d54-761b-4968-8511-ad5ff17c8c8c)

Reply:
and yes I have tested it